ios/ScannerNetwork.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ScannerNetwork.swift
+//  magnet
+//
+//  Created by Wilson Page on 11/05/2016.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+import Foundation
+
+@objc(ScannerNetwork) class ScannerNetwork: NSObject, NSNetServiceBrowserDelegate {
+  let MDNS_SERVICE_TYPE = "_http._tcp."
+  var browser: NSNetServiceBrowser!
```

Comment:
nit: browser we should rename it to serviceBrowser or something similar. It could be confusing.